client/idm/console/src/main/java/org/apache/syncope/client/console/status/ReconTaskPanel.java

```diff
@@ -143,7 +143,7 @@ protected Iterator<String> getChoices(final String input) {
                                     ? RealmsUtils.buildRootQuery()
                                     : RealmsUtils.buildKeywordQuery(input)).getResult())
                             : List.<RealmTO>of()).stream().
-                            map(RealmTO::getFullPath).collect(Collectors.toList()).iterator();
+                            map(RealmTO::getFullPath).toList().iterator();
```

Comment:
Why changing to `.toList().iterator()` and not directly `.iterator()`?

Reply:
Good point, thank you. I have revised all instances of this.